GCC 4.8.4 fails to build libva

CC       va_fglrx.lo                                                                                                        
va_fglrx.c: In function 'VA_FGLRXGetClientDriverName':                                                                        
va_fglrx.c:243:33: error: 'ADL_Main_Control_Destroy' may be used uninitialized in this function [-Werror=maybe-uninitialized] 
         ADL_Main_Control_Destroy();                                                                                          
                                 ^                                                                                            
cc1: all warnings being treated as errors                                                                                     
make[3]: *** [va_fglrx.lo] Error 1

--- a/configure.ac
+++ b/configure.ac
@@ -301,9 +301,6 @@ if test "$USE_DRM:$USE_X11:$USE_WAYLAND" = "no:no:no"; then
     AC_MSG_ERROR([Please select at least one backend (DRM, X11, Wayland)])
 fi
 
-AC_SUBST([AM_CFLAGS], ["-Wall -Werror"])
-AC_SUBST([AM_CXXFLAGS], ["-Wall -Werror"])
-
 AC_OUTPUT([
     Makefile
     doc/Makefile

